redis_session.c

```diff
@@ -581,6 +581,59 @@ redis_session_key(redis_pool_member *rpm, const char *key, int key_len, int *ses
     return session;
 }
 
+/* {{{ PS_CREATE_SID_FUNC
+ */
+PS_CREATE_SID_FUNC(redis)
+{
+    int retries = 3;
+#if (PHP_MAJOR_VERSION < 7)
+    char *sid;
```

Comment:
You can declare `sid` variable inside `while` loop

Reply:
Fixed